source/draft_api.rst

```diff
@@ -223,7 +258,9 @@ Response body:
       "issue_uri":"https://github.com/osu-cass/whats-fresh-api/issues/56",
       "date_worked":null,
       "created_at":2014-09-18,
-      "updated_at":null
```

Comment:
Lets make it clear, and no, users should not be able to modify updated or created dates